packages/web-scripts/README.md

```diff
@@ -93,6 +93,68 @@ These parallel builds are set up to share resources and work efficiently.
 
 If you need to use Babel for some reason, that's ok! Simply use babel directly instead of using `web-scripts build`. Teams inside Spotify mix and match which scripts they use to serve their needs. In many cases, `tsc` is all you need and is lighter and simpler to set up.
 
+### Setting up CI publishing (Travis CI)
+
+The following steps should be from your local repository folder.
+
+(Optional but probably mandatory): Visit https://travis-ci.com/account/repositories and click "Sync Account" otherwise the Travis CLI may not be able to register your ENV vars later. 
+
+1) Add a basic .travis.yaml. You probably want something like:
+
+```yml
+language: node_js
+node_js:
+- '8'
+- '10'
+- '12'
+branches:
+  only:
+  - master
+cache:
+  yarn: true
+  directories:
+  - node_modules
+before_install:
+- curl -o- -L https://yarnpkg.com/install.sh | bash -s
+- export PATH="$HOME/.yarn/bin:$PATH"
+script:
+- yarn lint
+- yarn test
+```
+
+
+2) Append a "release" stage to the `jobs:` that invokes `web-scripts release`:
+
+```
+jobs:
+  include:
+    - stage: release
+      node_js: lts/*
+      script: skip # do not run tests again
+      deploy:
+        provider: script
+        skip_cleanup: true
+        script:
+          - npx web-scripts release # or yarn release if you defined it in your package.json scripts
```

Comment:
does this need to be `@spotify/web-scripts`

Reply:
I think it technically does _not_, since npx will look in local node_modules/.bin first. But it might be safer to enforce both that and `--no-install`. I'll update.

Reply:
Instead I went with `yarn`. Yarn will look in node_modules/.bin, and error if the bin is not found. It only works with the bin name, not the scope prefix.